### ISSUE TITLE
Update to support setup.cfg and pyproject.toml

### ISSUE THREAD

**Issue by gsheni** [FeatureLabs/gh-action-pypi-upload#5]
- With the [new](https://github.com/alteryx/featuretools/pull/1941) pyproject.toml and setup.cfg, there is a different way to upload wheel files:
- Looking [here](https://setuptools.pypa.io/en/latest/build_meta.html?highlight=python%20-m%20build):
  - I know for a fact that the following steps work for releasing and uploading:
```shell
python -m pip install --upgrade build
python -m build
tar -zxvf "dist/featuretools-${VERSION}.tar.gz"
mv "featuretools-${VERSION}" unpacked_sdist\
pip install twine --upgrade
twine upload dist/* -r local --verbose --non-interactive --skip-existing
```
- I imagine we will have to modify the above steps

**Comment by gsheni** [comment]
The following change would need to happen (we may want to make sure this works on TestPyPI):
- https://github.com/FeatureLabs/gh-action-pypi-upload/blob/main/upload.sh#L51
That command is now `python -m build`

**Comment by gsheni** [comment]
Also this line:
- https://github.com/FeatureLabs/gh-action-pypi-upload/blob/main/upload.sh#L8
- We would need to update that. One way is to do the following :
```shell
FT_VERSION := $(shell grep '__version__\s=' featuretools/version.py | grep -o '[^ ]*$$')
```
- Another way is to do the following:
```shell
python -c 'import featuretools; print(featuretools.__version__)'
```

**Comment by thehomebrewnerd** [comment]
Closed by #6
